codechecker_lib/analyzers/result_handler_plist_to_db.py

```diff
@@ -33,6 +34,133 @@ def __init__(self, buildaction, workspace, run_id):
         super(PlistToDB, self).__init__(buildaction, workspace)
         self.__run_id = run_id
 
+    def __store_bugs(self, files, bugs, connection, analisys_id):
+        file_ids = {}
+        # Send content of file to the server if needed
+        for file_name in files:
+            file_descriptor = connection.need_file_content(self.__run_id,
+                                                           file_name)
+            file_ids[file_name] = file_descriptor.fileId
+
+            # sometimes the file doesn't exist, e.g. when the input of the
+            # analysis is pure plist files
+            if not os.path.isfile(file_name):
```

Comment:
This intended to cover the case when the source files from which the plists were generated are not in the file system. I don't know if there is any use case when the plists are separated from the code base. But what else would "parse from plist" functionality be used for? Although debug messages for non-existing source files is a good idea.